app/[docs_id]/chatForm.tsx

```diff
@@ -31,22 +31,19 @@ export function ChatForm({ documentContent }: { documentContent: string }) {
   return (
     <>
       {isFormVisible && (
-      <form className="border border-2 border-secondary shadow-xl p-6 rounded-lg bg-base-100" style={{width:"100%", textAlign:"center", boxShadow:"-moz-initial"}} onSubmit={handleSubmit}>
-          <h2 className="text-xl font-bold mb-4 text-left relative -top-2 font-mono h-2">
-            AIへ質問
-          </h2>
-        <div className="input-area" style={{height:"80px"}}>
+      <form className="border border-2 border-secondary shadow-md rounded-lg bg-base-100" style={{width:"100%", textAlign:"center", boxShadow:"-moz-initial"}} onSubmit={handleSubmit}>
+        <div className="input-area" style={{height:"90px", margin:"10px"}}>
             <textarea
-              className="textarea textarea-white textarea-md"
+              className="textarea-ghost textarea-white textarea-md"
```

Comment:
```suggestion
              className="textarea textarea-ghost textarea-md"
```

`textarea-white` はdaisyuiに存在しないですね
逆に `textarea` を指定しないといけないはずです
https://daisyui.com/components/textarea/
そうするとdaisyui側でデフォルトのpaddingとかが追加されると思うので、margin:10px とかを消す必要があるかも